serum-390/soen390-team16-godzilla
# Finished the about us page

# Description

Added a markdown rendering component using `react-markdown` . 

Both Desktop and Mobile versions are included . Desktop has a paper component as the background, while mobile simply renders the markdown to the page without a paper

## Link to work item

Closes #152

## How to Test

1. Find the aboutUs.md file in  **components\markDown\aboutUs.md**
2.  Add/ delete some text
3. reload the react app and see if the changes apply
